Formula/perl.rb

```diff
@@ -81,12 +72,13 @@ def caveats; <<-EOS.undent
 
     You can set that up like this:
       PERL_MM_OPT="INSTALL_BASE=$HOME/perl5" cpan local::lib
-      echo 'eval "$(perl -I$HOME/perl5/lib/perl5 -Mlocal::lib)"' >> #{shell_profile}
+      echo 'eval "$(perl -I$HOME/perl5/lib/perl5 -Mlocal::lib)"' >> #{Utils::Shell.profile}
```

Comment:
[It seems to be pending](https://github.com/Homebrew/brew/blob/744e7dd1907aaf6493e6e4622a5997b562bfbf17/Library/Homebrew/compat/utils.rb#L2).